linker-interface/shared/src/main/scala/org/scalajs/linker/interface/ModuleSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker.interface
+
+import org.scalajs.linker.interface.unstable.ModuleSpecImpl.SelectorImpl
+
+/** Module output specification. */
+final class ModuleSpec (
```

Comment:
I have also considered to make a `ModuleSpec` a list inherently (like in an earlier proposal). However, it seems that with the fully automatic module tracking, the input `ModuleSpec` is unlikely to turn into something more automatic.